src/main/haskell/language-kore/src/Data/Kore/Parser/ParserUtils.hs

```diff
@@ -11,6 +11,7 @@ Helper tools for parsing Kore. Meant for internal use only.
 -}
 module Data.Kore.Parser.ParserUtils where
 
+import           Data.Functor                     (($>))
```

Comment:
I'm not convinced `$>` always adds clarity, but it looks ok in in the suggestions you've made